test/files/neg/case-collision-multifile/one.scala

```diff
@@ -1,2 +1,2 @@
-// scalac: -Werror
+// scalac: -Werror -Ybackend-parallelism 1
```

Comment:
opportunity for `using options`, which is not a big deal to an `haut dawg` such as yourself.

Setting to 1 is to avoid re-ordering of `hotDog` and `HotDog` in message?